### PR TITLE
Allow write-only in column definition

### PR DESCRIPTION
**Pull request by dukenguyenxyz**

### CODE REVIEW

File: src/clear/model/modules/has_columns.cr [hunk 1]
```diff
@@ -146,7 +146,10 @@ module Clear::Model::HasColumns
   #   when instantiating or updating a new model from json through `.from_json` methods from
   #   the `Clear::Model::JSONDeserialize` module.
   #
-  macro column(name, primary = false, converter = nil, column_name = nil, presence = true, mass_assign = true)
+  # * `write_only : Bool (default = true)`: same as `ignore_serialize`: turn on/ off serialization
+  #   of a field when doing `.to_json` on the model
+  #
```
Comment (caspiano) [inline src/clear/model/modules/has_columns.cr]: This doesn't make sense to me. `write_only` implies you cannot read the value. No point overloading/changing the semantics of `ignore_serialize` if that's what it implies.

Comment (dukenguyenxyz) [reply src/clear/model/modules/has_columns.cr]: This is mainly for OpenAPI and partially for anything else. `write-only` is OpenAPI's terminology.

Comment (caspiano) [reply src/clear/model/modules/has_columns.cr]: In that case, better to do it where there's an interface between the two libraries, or via tags that don't impact serialisation.
These are two separate concerns.

A general user of `clear` probably doesn't want to buy into OpenAPI tags that don't particularly make sense in the context of the library for their use case

**Review by caspiano — APPROVED** [review]

**Comment by caspiano** [comment]
@dukeraphaelng if you update the PR body to outline this work, it may help with batching together changes for making a PR to the main `clear` repo

**Comment by dukenguyenxyz** [comment]
Closing due to this already being handled in `place-labs/openapi-generator`

**Pull request closed without merging**